server/src/main/java/io/littlehorse/common/util/RocksConfigSetter.java

```diff
@@ -65,6 +66,9 @@ public void setConfig(final String storeName, final Options options, final Map<S
         tableConfig.setBlockSize(BLOCK_SIZE);
         options.setTableFormatConfig(tableConfig);
 
+        // Compression
+        options.setCompressionType(CompressionType.ZSTD_COMPRESSION);
```

Comment:
Hmm. Thought this was already in `master`?

Reply:
You must rebase or sync with master